### ISSUE TITLE
LoKr lycoris 0.1.6 vs 0.1.7+

### ISSUE THREAD

**Issue by ckmlai** [KohakuBlueleaf/LyCORIS#86]
Hi there!

I've been using LoKr for training concepts and styles for quite a while, but from my testing, every version of the lycoris repo after v0.1.7 seems to produce worse results when paired with adaptive optimizers like Prodigy and DAdaptation.
From the data I gathered, the estimated d*lr for v0.1.6 seems to not get past 1e-5 ~ 1e-4 while v0.1.7.x and v0.1.8.x seem to go as high as 0.001. While an aesthetically accurate evaluation would be pretty hard to do, the results from doing x/y grids are noticeably worse for the loras trained with v0.1.7+

![wandb subject 1](https://github.com/KohakuBlueleaf/LyCORIS/assets/140862089/bf634b3f-2204-44d6-9399-bec70f924011)

![wandb subject 2](https://github.com/KohakuBlueleaf/LyCORIS/assets/140862089/0e49038a-27dc-404b-abc5-565e2e0859bc)

Both loras were trained using almost the same parameters, save for the adjustments made to the cosine annealing scheduler args to match the training steps corresponding to the one estimated to their datasets.

Is this intended and I should try a different parameters for the scheduler and optimizer?

**Comment by KohakuBlueleaf** [comment]
@ckmlai Sorry for that!
We are doing experiments on some different weight initialize method
I will make a network_args for specifiying the method you want to used in 1.8.x

**Comment by ckmlai** [comment]
@KohakuBlueleaf no problem! And thank you for the amazing working as always!
